apps/judicial-system/web/src/routes/Court/Indictments/Summary/Summary.tsx

```diff
@@ -83,6 +83,22 @@ const Summary: FC = () => {
     [[] as CaseFile[], [] as CaseFile[]],
   )
 
+  const getRulingDecisionTagColor = (
```

Comment:
I know this doesn't quite fit our handling of tags in other places, but as this is only used here for now and also I think we need to rethink the Tag handling as it's gotten very complicated with time, I wanted to just have this be simple and we can then revamp when needed.